### DISCUSSION TITLE
[Question]: Access clip without Authentification error

### DISCUSSION THREAD

**Discussion by Nonobis** [blakeblackshear/frigate#13249] [Ask A Question]
### What is your question:

Hello,

I am trying to configure Frigat-Notify along my Frigate.
All is working on my ntfy notification but one link. The direct link to the video clip.

https://ntfy.xxx/api/events/1724251555.455906-1aeub7/clip.mp4

I have an "Authentification Required" when i click on th elink.
I use NPM as reverse proxy. Do i need to exclude a route on the config or maybe i need to modify the url send from the notification ?

I have also tryied :

https://login@password:ntfy.xxx/api/events/1724251555.455906-1aeub7/clip.mp4

![image](https://github.com/user-attachments/assets/aba18145-749a-4aad-a48c-699ec800eb7f)

but i have the same error

Any idea ?

**Comment by NickM-27** [comment]
It's most likely accessing via a browser tab that doesn't have the authentication cookie saved, there's no great way to adjust this link if it is being opened in a browser that you aren't logged in to frigate in

If you can install frigate as a PWA then it's easiest to just link directly into the UI.

**Comment by Nonobis** [comment]
hum ok, i will try to login to manually to frigate before clicking to the link... i can maybe also attach the video clip to the notification.
